packages/yourloops/components/statistics/patient-statistics.tsx

```diff
@@ -48,15 +47,29 @@ export const PatientStatistics: FunctionComponent<PropsWithChildren<PatientStati
   const cbgStatType: CBGStatType = bgSource === DatumType.Cbg ? CBGStatType.TimeInRange : CBGStatType.ReadingsInRange
   const numberOfDays = TimeService.getNumberOfDays(dateFilter.start, dateFilter.end, dateFilter.weekDays)
   const cbgSelected = bgSource === DatumType.Cbg
-  const { sensorUsage, total } = GlycemiaStatisticsService.getSensorUsage(medicalData.cbg, numberOfDays, dateFilter)
-  const sensorUsageData = {
+  const bgUnits = MGDL_UNITS || MMOLL_UNITS
+  const theme = useTheme()
+  const {
+    sensorUsage,
+    totalUsage
+  } = GlycemiaStatisticsService.getSensorUsage(medicalData.cbg, numberOfDays, dateFilter)
+  const {
+    glucoseManagementIndicator,
     total,
+    insufficientData
+  } = GlycemiaStatisticsService.getGlucoseManagementIndicatorData(medicalData.cbg, bgUnits, dateFilter)
+  const glucoseManagementIndicatorData = {
+    glucoseManagementIndicator,
+    totalGMI: total,
+    title: bgSource
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (blocking):** ‏`title` is given the value `bgSource` which is in fact a `BgType`? What? Can we at least reame the prop `bgSource` to `bgType`?